test/users.js

```diff
@@ -0,0 +1,142 @@
+const chai = require('chai'),
+  server = require('./../app'),
+  dictum = require('dictum.js'),
+  sessionManager = require('./../app/services/sessionManager'),
+  should = chai.should();
+
+describe('users', () => {
+  describe('users sessions', () => {
+    it('should fail because of invalid email', done => {
+      chai
+        .request(server)
+        .post('/users/sessions')
+        .send({ email: 'invalid', password: '12345678' })
+        .catch(err => {
+          err.should.have.status(400);
+          err.response.should.be.json;
+          err.response.body.should.have.property('error');
+        })
+        .then(() => done());
+    });
+  });
+
+  describe('users create', () => {
+    it('creation OK', done => {
```

Comment:
Keep the test descriptions with clear semantics, for example, `it('should be successful' , .....`